entries/b/bankofamerica.com.json

```
@@ -4,11 +4,12 @@
     "url": "https://www.bankofamerica.com/",
     "tfa": [
       "sms",
-      "email"
+      "email",
+      "custom-hardware"
```

```suggestion
      "u2f"
```
Based on the link provided in the issue, U2F is supported.